docs/tech-reference/yum-plugins.rst

```diff
@@ -596,9 +596,10 @@ Optional Configuration Parameters
  full path to the certificate.
 
 ``gpgkey``
- GPG key used to sign RPMs in this repository. This key will be made available
- to consumers to use in verifying content in the repository. The value to this
- option must be the full path to the GPG key file.
+ File containing GPG keys used to sign RPMs and metadata in this repository.
```

Comment:
This is a little confusing. The "File containing GPG keys used to sign RPMs" is just used for clients to validate GPG signatures, but you can read it differently and it reads as though Pulp is doing actual GPG signing of RPMs. However, the metadata is actually signed by Pulp in this PR.